cartridge/feedback.lua

```diff
@@ -1,18 +1,68 @@
+local fio = require('fio')
 local log   = require('log')
 local json  = require('json')
 local fiber = require('fiber')
 local http  = require('http.client')
 
 local PREFIX = "feedback_daemon"
 
+local function parse_lua_file(name)
+    local state = {}
+    local eval = loadfile(name, 't', state)
+    pcall(eval)
+
+    return state
+end
+
+local function parse_manifest(filename)
+    local manifest = parse_lua_file(filename)
+    local deps = {}
+
+    for module, versions in pairs(manifest.dependencies or {}) do
+        for version, _ in pairs(versions) do
+            if deps[module] ~= nil then
+                break -- Decide how to deal with duplicates
```

Comment:
Do you have any ideas?

Reply:
I'm not sure we can anything with it by just parsing `manifest` file. I think, we may just assume, there is consistent `.rocks/` directory and just take the first version.

More reliable approach needs to examine `.rocks/` structure.

Reply:
Decided to take only first version, since it is difficult to detect which version is actually used and requires parsing a file tree